relays/utils/src/relay_loop.rs

```diff
@@ -24,6 +24,9 @@ use substrate_prometheus_endpoint::{init_prometheus, Registry};
 /// Default pause between reconnect attempts.
 pub const RECONNECT_DELAY: Duration = Duration::from_secs(10);
 
+/// Generic relay loop metrics that may be reused by any loop.
+pub type GenericLoopMetrics = LoopMetrics<(), (), ()>;
```

Comment:
Looks like this alias isn't used anythere, so why have it at all. We can always add it, once it is acutally used.